core-sitemaps.php

```diff
@@ -19,11 +21,12 @@
 
 const CORE_SITEMAPS_POSTS_PER_PAGE = 2000;
 
+require_once __DIR__ . '/inc/class-sitemaps.php';
 require_once __DIR__ . '/inc/class-sitemaps-provider.php';
 require_once __DIR__ . '/inc/class-sitemaps-index.php';
 require_once __DIR__ . '/inc/class-sitemaps-pages.php';
 require_once __DIR__ . '/inc/class-sitemaps-posts.php';
 require_once __DIR__ . '/inc/class-sitemaps-registry.php';
+require_once __DIR__ . '/inc/registration.php';
```

Comment:
I don't fully understand why this is needed separately from registry and provider?

Reply:
Well the idea was to give an example for others to reuse, but it's not that complex and on reviewing your comment I've changed it so it's bundled with the filter as the default value.